server/lib/adapters/mysql.js

```diff
@@ -42,6 +43,23 @@ export function validateResult(result) {
 }
 
 /**
+ *
+ * @param error from database connector
+ * @returns {{errors: Array}}
+ */
+
+export function checkForError(error) {
+  if (error && (error.code === "ER_PARSE_ERROR" || error.code === "ER_NO_SUCH_TABLE")) {
+    return { message: `Invalid Syntax: ${_.get(error, "sqlMessage", "")}` };
```

Comment:
hum... if `ER_NO_SUCH_TABLE ` I would not say that it's an Invalid Syntax error...